### ISSUE TITLE
Feature Request: Add dynamic sizing and color change

### ISSUE THREAD

**Issue by Tech-Helper503** [nicolasdejong/ShakeMouseToLocate#5]
I am using a custom pointer scheme, could you make it adapt to the scheme and slowly size it like on MacOS

**Comment by nicolasdejong** [comment]
No. Sorry, but AutoIt scripts are meant for small simple stuff. The functionality here is limited and the script has become quite long already. I have been experimenting with c# in the past and that would make things simpler, but since then I have moved almost 100% to Mac (except if I get hired for something where I'm not allowed to use my own tools). I prefer not to maintain more complex software for a platform I hardly use anymore.

Perhaps later, when I add a few cursors to choose from (see #4 ) you have a better match for your custom pointer scheme.

**Comment by Tech-Helper503** [comment]
I think I can make a fork of this project with more complex features like less sensitivity, an animation to change the size of the mouse the more you shake it(like on mac) and customizable color schemes

**Comment by nicolasdejong** [comment]
If you have the time for it, please do.  I built on the work of Spinal Cord. If you build on this, each version is an improvement over the other.
